packages/app/src/routers/ViewRouter/ViewRouter.tsx

```diff
@@ -70,6 +82,7 @@ export const ViewRouter = ({ accessToken }: ViewRouterProps) => {
         hideIModelActions={["view"]}
       />
       <View path="project/:projectId/imodel/:iModelId" />
+      <View path="project/:projectId/imodel/:iModelId/version/:versionId" />
```

Comment:
need to pass in accessToken here

Reply:
Yup, how is that even sometime working in my tests ???

Reply:
Added, fixed possible re-render while waiting on changesetId fetch results.